src/main/java/nl/b3p/tailormap/api/persistence/GeoService.java

```diff
@@ -178,12 +184,20 @@ public String getUrl() {
     return url;
   }
 
-  /** Sets the url after sanitising (removing unwanted parameters). */
+  /** Sets the url after sanitizing (removing unwanted parameters). */
```

Comment:
```suggestion
  /** Sets the url after sanitising (removing unwanted parameters). */
```
Sanitise is the preferred spelling in British English, Australian English, and everywhere else.